components/brave_wallet/browser/brave_wallet_prefs.cc

```diff
@@ -42,7 +38,18 @@ base::Value GetDefaultUserAssets() {
   bat.SetKey("decimals", base::Value(18));
   bat.SetKey("visible", base::Value(true));
   bat.SetKey("logo", base::Value("bat.svg"));
-  user_assets_list->Append(std::move(bat));
+
+  // Show ETH and BAT by default for mainnet, and ETH for other known networks.
+  base::Value user_assets_pref(base::Value::Type::DICTIONARY);
+
+  std::vector<std::string> network_ids = brave_wallet::GetAllKnownNetworkIds();
```

Comment:
This includes localhost chain, let me know if we want to exclude it instead.

Reply:
include is fine